src/lib/utils.js

```diff
@@ -1,4 +1,4 @@
-import launchDarklyBrowser from "ldclient-js";
+import * as launchDarklyBrowser from "ldclient-js";
```

Comment:
This fixes issue #89

Reply:
reverting for now. it created some issues with the tests

Reply:
I think I'm fine with this change, though I'm a little ambiguous about what bug we're fixing.  All I can google is that flowtype has misconfigured their definition of modules as solely commonJS, ergo everybody else has to as well.  That doesn't seem to be a good reason to change, imo.

Still fine, there's no harm here.  But next time we ought to stick to one feature/issue per patch.